### PR TITLE
Just adding a line in Earth.txt

### PR DESCRIPTION
**Pull request by MKourdourli**
Is it ok to add this line in Earth.txt ?

### CODE REVIEW

**Comment by AlixYAN** [comment]
Hey Mehdi :D

**Comment by MKourdourli** [comment]
Hello Alix! No Rainbow parrot here :(

File: mehdi_Earth.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Earth is the third planet from the Sun and the only astronomical object known to harbor life. About 29% of
+Earth's surface is land consisting of continents and islands and somewhat interesting people. The remaining 71% is covered with water,
+mostly by oceans but also by lakes, rivers, and other fresh water, which together constitute the hydrosphere.
+Much of Earth's polar regions are covered in ice. Earth's outer layer is divided into several rigid tectonic
+plates that migrate across the surface over many millions of years. Iva is adding a sentence in this file. Earth's interior remains active with a
+solid iron inner core, a liquid outer core that generates Earth's magnetic field, and a convecting mantle
+that drives plate tectonics.
+
+Just adding a line here.
```
Comment (PabloRdrRbl) [inline mehdi_Earth.txt:9]: Look, I can create comments to review your code and ask you for changes.

**Review by PabloRdrRbl — APPROVED** [review]
Now, I approve.

**Pull request merged**